api/onboardingpb/json/onboardingpb.json

```diff
@@ -0,0 +1,192 @@
+{
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "schemes": [
+    "https",
+    "http"
+  ],
+  "swagger": "2.0",
+  "info": {
+    "title": "PMM Platform API",
+    "version": "v1"
+  },
+  "paths": {
+    "/v1/onboarding/tips": {
+      "post": {
+        "description": "Method marks user tip is done. Only works for user tip, not for system tips.",
+        "tags": [
+          "TipService"
+        ],
+        "summary": "Complete user tip",
+        "operationId": "CompleteUserTip",
+        "parameters": [
+          {
+            "description": "Request message for CompleteUserTip method.",
+            "name": "body",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "description": "Request message for CompleteUserTip method.",
+              "type": "object",
+              "properties": {
+                "tipId": {
+                  "description": "The ID of the tip to retrieve.",
+                  "type": "integer",
+                  "format": "int32",
+                  "x-order": 0
+                },
+                "userId": {
+                  "description": "user id which completes user tip.",
+                  "type": "integer",
+                  "format": "int32",
+                  "x-order": 1
+                }
+              }
+            }
+          }
+        ],
+        "responses": {
```

Comment:
404?

Reply:
how can we do that? it's generated automatically and if you take a look on other swagger files it's always only 200 or unexpected error.